### ISSUE TITLE
[Request] Support for Tesoro RGB and Thermaltake RGB Keyboards

### ISSUE THREAD

**Issue by Xelus22** [SpoinkyNL/Artemis#47]
Would love for this program to support these keyboards. I have contacted Tesoro and Thermaltake just through public 'contact us' box on their websites for the API for you guys to use.
Edit: im a nobody 

Thanks,
Xelus

**Comment by Xelus22** [comment]
Hey guys, great news. I got an email back from thermaltake. Below is copy+paste of email.

Hi [my name],
 
Thanks for getting in touch with us at Thermaltake Australia and New Zealand. We've notified our Tt eSPORTS HQ software development team in Taiwan regarding your API request for our Poseidon Z RGB Mechanical Gaming Keyboard, and we're now currently awaiting their response. We'll let you know of any updates to this status as it's processed.
 
Great work on the open-source project, it looks like it's a fantastic interface to use and operate. Best of luck with the continuation of it's development, and once again thanks for getting in touch with us. Hopefully we'll have some good news for you soon. :)
 
Kind regards,
Jeremy Tate
